lib/auth_app/accounts/user.ex

```diff
@@ -8,27 +8,20 @@ defmodule AuthApp.Accounts.User do
     field :hashed_password, :string, redact: true
     field :current_password, :string, virtual: true, redact: true
     field :confirmed_at, :utc_datetime
+    field :authenticated_at, :utc_datetime, virtual: true
```

Comment:
I believe this should be stored in the session, not in the user, otherwise the actual user in one machine will give sudo access for an attacker in another machine.

Reply:
Well actually it is, this is only a virtual field populated from the session token; but when we do sudo mode with redirects we probably don't need it at the user

Reply:
Doh, it is virtual, sorry, ignore me!